app/controllers/dish_ingredients_controller.rb

```diff
@@ -0,0 +1,13 @@
+class DishIngredientsController < ApplicationController
+
+  def new
```

Comment:
Naming this action `create` would be more conventional, as this is the action where you're actually creating a resource (#new is typically for rendering a form but in this case the form is rendered on a dish show page)